packages/kit/src/views/FirmwareUpdate/components/FirmwareChangeLogView.tsx

```
@@ -6,6 +6,7 @@ import { StyleSheet } from 'react-native';
 import {
   Accordion,
   Icon,
+  IStackProps,
```

ESLint

[error] Member 'IStackProps' of the import declaration should be sorted alphabetically.

(sort-imports)